doc/api/index.md

```diff
@@ -1478,6 +1480,27 @@ it was called:
 }
 ```
 
+<a name="meth-disableVideoTrack"></a>
+### disableVideoTrack #####################################################
+
+_return value_: ``void``
+
+Disable the video track adaptation.
+
+This permits to play in **Audio only** mode.
+
+> Will enter in `RELOADING` state for a short period.
+
+<a name="meth-disableAudioOnly"></a>
+### disableAudioOnly #####################################################
+
+_return value_: ``void``
+
+Disable the **Audio only** mode, by switching from audio adaptation only to video/audio adaptations.
+
+This permits to play in **normal** mode with video.
```

Comment:
**normal** mode is a weird way to put it.
Just saying that it disable audio only mode is enough I think